docs/EdgeCases.md

```diff
@@ -1,3 +1,11 @@
+# Edge Cases
+This section describes some edge-cases and scenarios where behavior is not ideal or is unknown.
+
+## Scaling Provider replicas down
+It has been observed that when pods are running across multiple provider instances, scaling the number of providers down doesn't reschedule those pods onto other remaining provider instances.  For instance if 99 pods are running across 3 provider instances in a StatefulSet (33 pods per provider), and the provider instances are scaled from 3 to 2, the 33 pods it was responsible for appeared to "wait" for the provider instance to return (a sort of "stickiness" of pods to provider instances).  This _may_ have been a configuration-related behavior in the particular cluster it was observed in and the experiment should be repeated.
```

Comment:
Instead can we add a note to say how to drain the pods from one of the VK and then delete the VK node.

Reply:
Done!  https://github.com/awslabs/aws-virtual-kubelet/blob/7c3d96f1e1b822b63c9457336456c6dc2d98435f/docs/EdgeCases.md#scaling-provider-replicas-down and https://github.com/awslabs/aws-virtual-kubelet/blob/7c3d96f1e1b822b63c9457336456c6dc2d98435f/docs/Cookbook.md#scale-vk-statefulset-replicas-down